tests/test_dryrun.py

```diff
@@ -34,6 +36,18 @@ def test_sql_file_invalid(self, tmp_path):
         dryrun = DryRun(str(query_file))
         assert dryrun.is_valid() is False
 
+    def test_view_file_valid(self):
+        view_file = (
+            TEST_DIR
+            / "sql"
+            / "moz-fx-data-shared-prod"
+            / "messaging_system"
+            / "cfr_users_last_seen"
+            / "view.sql"
+        )
```

Comment:
We generally haven't used the `tests/sql` directory to store test assets like this. Could put these in a `test_fixture` dataset instead e.g. `moz-fx-data-shared-prod/test_fixtures/cfr_users_last_seen`, or point directly to the view in the top-level `sql` directory.

Even better would be to simplify the queries even further and write them directly into a temporary file here e.g.

```python
def test_view_file_valid(self, tmp_path):
    sql = """CREATE OR REPLACE
    -- add the minimal amount of query necessary to reproduce the error
    ...
    """
    path = (tmp_path / "view.sql").write_text(sql)
    assert DryRun(str(path)).is_valid()
```

And so forth.